units/src/amount/tests.rs

```diff
@@ -84,21 +96,18 @@ fn test_amount_try_from_signed_amount() {
 
 #[test]
 fn mul_div() {
-    let sat = Amount::from_sat;
-    let ssat = SignedAmount::from_sat;
```

Comment:
I don't understand this commit.  Why not just change to the following?:

```rust
    let sat = Amount::from_sat_unchecked;
    let ssat = SignedAmount::from_sat_unchecked;
```

Reply:
You mean the commit that adds the macro? I rekon it would be better to panic in the macro when we enforce the invariant so that we do no accidentally have tests that have values outside of the valid range after we start enforcing `MAX_MONEY`.

I should have written that in the commit log.